stack-easybib/recipes/deploy-rdstest.rb

```diff
@@ -0,0 +1,25 @@
+get_apps_to_deploy.each do |application, deploy|
+  case application
+  when 'rds_test'
+    unless allow_deploy(application, 'rds_test', %w(nginxphpapp))
+      Chef::Log.info("stack-easybib::deploy-rdstest - #{application} skipped")
+      next
+    end
+  else
+    Chef::Log.info("stack-easybib::deploy-rdstest - #{application} skipped")
+    next
+  end
+
+  Chef::Log.info("stack-easybib::deploy-rdstest - Deployment started as #{deploy[:user]}:#{deploy[:group]}")
+
+  easybib_deploy application do
+    deploy_data deploy
+  end
+
+  cron_d 'rds_test' do
```

Comment:
@baccenfutter maybe touch base with @glaubinix and @naderman – I think the idea was to run the test script via FPM — so more like "inside" one of the apps. This would use the cli again?